### DISCUSSION TITLE
Sponsorship stuck at “Complete the required documentation to receive payouts”

### DISCUSSION THREAD

**Discussion by MrMarble** [community/community#23428] [New to GitHub]
<p>Dear all,</p>
<p>I applied for GitHub sponsorship program, however I can’t finish the application since it’s stuck at:</p>
<p><strong>Complete the required documentation to receive payouts</strong></p>
<p>I completed all docs at Stripe and this status is still there for a few days now.</p>
<p>Please advise?</p>
<p>Thank you,<br>
Alvaro</p>

**Comment by totov** [comment]
<p>I’m getting the exact same issue. Were you able to resolve this issue or is it still broken for you?</p>

**Comment by MrMarble** [comment, marked as answer]
<p>Yes it resolved by itself, it took like 3 weeks for it to advance to the next step but then everything else was done in less than an hour <a class="mention" href="/totov">@totov</a></p>

**Comment by totov** [comment]
<p>Thanks. I had to contact GitHub (opening a ticket on the <a href="https://support.github.com/tickets/personal/0" rel="noopener nofollow ugc">support site</a>) to get this resolved as Stripe was out of sync with GitHub which had stalled my application. Sorted now.</p>
